doc/TROUBLESHOOT.md

```diff
@@ -4,24 +4,51 @@ The text below is under construction.
 
 TODO: separate information for tests and applications.
 
-# Memory Leak Troubleshooting
+# Troubleshoot Memory Leaks
 
-This page describes how to troubleshoot memory leaks. See leak detection guidance [here](DETECT.md).
+This page describes how to troubleshoot memory leaks. See other information on memory leaks [here](../README.md).
 
-## Collect callstack
+To understand root cause of a memroy leak, gather additional information.
```

Comment:
sp: memory

To understand the root cause of a memory leak, you may need to gather additional information.